Perlang.Stdlib/Stdlib/Argv.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections.Concurrent;
+using System.Collections.Immutable;
+using Perlang.Attributes;
+using Perlang.Exceptions;
+
+namespace Perlang.Stdlib
+{
+    [GlobalClass]
+    public static class Argv
+    {
+        private static ConcurrentQueue<string> remainingArguments;
+
+        [ArgumentsSetter]
+        public static void SetArguments(ImmutableList<string> arguments)
+        {
+            remainingArguments = new ConcurrentQueue<string>(arguments);
+        }
```

Comment:
I'm not incredibly proud of this approach. Global, mutable state is [clearly evil](https://medium.com/@piotr.chmielowski/singleton-is-not-an-anti-pattern-51392c3e508e). On the other hand, we don't support invoking instance-methods at the moment from Perlang code, and even if we would, we'd still need some form of "container" object to hold the `Argv` instance.

I'm willing to say we suffer with this deficiency for now. It's the best we can do at the moment, and it's at least thread-safe (even though popping command line arguments from multiple threads concurrently would be a rather absurd programming pattern. :see_no_evil:)